src/types.ts

```diff
@@ -8,9 +8,6 @@ import { createContext, JitContext } from 'tailwindcss/src/lib/setupContextUtils
 export interface JitState extends State {
   config: object;
   separator: string;
-  screens: string[];
-  variants: Record<string, string | null>;
-  jit: true;
```

Comment:
FYI I just added this since it must be true but nvm 😇

Reply:
It could be false when it’s processed by tailwind though. I don’t know how they deal with it internall :shrug:

Reply:
All fine 👍